src/web/components/Layout/Layout.js

```diff
@@ -47,10 +48,30 @@ const Title = styled.header`
 
 const Content = styled.section`
   grid-area: content;
+
+  color: white;
 `;
 
 function Layout() {
   const theme = useTheme();
+  const [settings, setSettings] = useState({});
+
+  useEffect(() => {
+    // eslint-disable-next-line
```

Comment:
Would be good if you could find a way to configure eslint so that you do not have to disable it every time you reference ipc.

Reply:
This is now done. You just have to pull it from the `window` object.